src/actions/groups.ts

```diff
@@ -253,6 +264,25 @@ export function getGroupsAssociatedToChannel(channelID: string, q = '', page = 0
             q,
             page,
             perPage,
+            filterAllowReferences,
         ],
     });
 }
+
+export function patchGroup(groupID: string, patch: GroupPatch): ActionFunc {
```

Comment:
Can you use the shorthand `bindClientFunc` for the body of this function because there's nothing unusual happening? See https://github.com/mattermost/mattermost-redux/blob/master/src/actions/schemes.ts#L62 as an example.